code/modules/mining/mint.dm

```diff
@@ -122,6 +122,19 @@
 				if (params["material_name"] == mat.name)
 					chosen = mat
 
+obj/machinery/mineral/mint/RefreshParts()
+
+	for(var/obj/item/stock_parts/matter_bin/B in component_parts) // Better matter bins increases the output of coins per cycle.
+	
+	for(var/obj/item/stock_parts/micro_laser/L in component_parts) // Better microlasers increase reduce the delay between production cycles.
+	
+	for(var/obj/item/stock_parts/manipulator/M in component_parts) // Better manipulators increase amount of coins produced per sheet.

```

Comment:
ah yes I too love infinite material duping

Reply:
Just make t4 100% effective.
And t1 25% effective.

Reply:
That'd be a good solution

Reply:
check the code, it doesn't do that 😏

Reply:
what?

Reply:
oh yeah i realize but the comment shows they want to add it